scripts/run_package_tests.sh

```diff
@@ -14,22 +14,21 @@ set -euo pipefail
 node --version
 
 # Install the shields.io dependencies.
-if [[ "$NODE_VERSION" == "v10" ]]; then
-    # Avoid a depcheck error.
-    npm ci --ignore-scripts
-else
-    npm ci
-fi
+npm ci
 
 # Run the package tests.
 npm run test:package
 npm run check-types:package
 
-# Delete the shields.io dependencies.
+# Delete the full shields.io dependency tree
 rm -rf node_modules/
 
+
 # Run a smoke test (render a badge with the CLI) with only the package
 # dependencies installed.
 cd badge-maker
+
+npm install # install only the package dependencies for this test
 npm link
 badge cactus grown :green @flat
+rm package-lock.json && rm -rf node_modules/ # clean up package dependencies
```

Comment:
This had me stumped for a bit. I think what was going on here is:
When we were doing the first `npm ci` under node 10, we were actually using an old version of NPM (5.x ?) which created a separate `node_modules/` under the `badge-maker/` dir. After dropping node 10, now the initial `npm ci` happens using a more recent version which _doesn't_ do this (everything goes into the root `node_modules/`) so then when we come to run this "test" we need to explicitly install (and clean up) in the package dir.

Reply:
I'm surprised by the need to cleanup here, but so long as we don't have a chance of false positives where deps are missing for the package but present in the server manifest, which seems to still hold true here